### PR TITLE
docs/concepts/aws.md: document TLS handshake errors in kube-apiserver

### LINKED ISSUE #438: Document TLS Handshake errors on AWS single master cluster
On apiserver you can see multiple TLS Handshake errors as reported in https://github.com/kinvolk/lokomotive/issues/278.

But this is cannot be fixed on AWS if the cluster has single node master. So document it as mentioned in: 

> For AWS, the TLS handshake errors cannot be resolved, as they are caused
> by AWS Load Balancer, which is configured to only TCP checks and in HTTP
> mode, it does not support authentication header, so we would have to
> remove --anonymous-auth=false kube-apiserver option, which we shouldn't
> do.

https://github.com/kinvolk/lokomotive/issues/278#issuecomment-614574108

cc: @invidian

### PR DESCRIPTION
**Pull request by invidian**
Closes #438.

Signed-off-by: Mateusz Gozdek <mateusz@kinvolk.io>

### CODE REVIEW

File: docs/concepts/aws.md [hunk 1]
```diff
@@ -28,3 +28,17 @@ spec:
     - ipBlock:
         cidr: 0.0.0.0/0
 ```
+
+## TLS handshake errors in kube-apiserver logs
+
+On AWS platform, you may see the following logs coming from `kube-apiserver` pods:
```
Comment (iaguis) [inline docs/concepts/aws.md:34]:
```suggestion
On the AWS platform, you may see the following logs coming from `kube-apiserver` pods:
```

File: docs/concepts/aws.md [hunk 2]
```diff
@@ -28,3 +28,17 @@ spec:
...
+
+```
+I0408 05:35:02.865305       1 log.go:172] http: TLS handshake error from 127.0.0.1:45332: read tcp 127.53.210.227:7443->127.0.0.1:45332: read: connection reset by peer
+I0408 05:35:12.865457       1 log.go:172] http: TLS handshake error from 127.0.0.1:45424: read tcp 127.53.210.227:7443->127.0.0.1:45424: read: connection reset by peer
+I0408 05:35:22.865279       1 log.go:172] http: TLS handshake error from 127.0.0.1:45516: read tcp 127.53.210.227:7443->127.0.0.1:45516: read: connection reset by peer
+```
+
+Those logs are harmless and are caused by AWS ELB probing `kube-apiserver` for availability. Unfortunately, AWS ELB does not support authentication for probe requests, so we must use TCP probes, which causes exactly those errors.
```
Comment (iaguis) [inline docs/concepts/aws.md:42]:
```suggestion
These errors are harmless and are caused by an AWS ELB probing `kube-apiserver` for availability. The probe succeeds but AWS ELBs do not support authentication for probe requests so `kube-apiserver` complains with error logs.
```

Comment (invidian) [reply docs/concepts/aws.md:42]: This does not seem logical to me to mention authentication if we change it like this.

Comment (invidian) [reply docs/concepts/aws.md]: Rephrased. Is it better now?

File: docs/concepts/aws.md (hunk 1 above)
Comment (invidian) [reply docs/concepts/aws.md]: Thanks.

**Pull request merged**